src/mpid/ch4/src/ch4_impl.h

```diff
@@ -128,21 +128,21 @@ static inline MPIR_Context_id_t MPIDIG_win_to_context(const MPIR_Win * win)
 }
 
 #undef FUNCNAME
-#define FUNCNAME MPIDI_CH4U_request_complete
+#define FUNCNAME MPIDIU_request_complete
 #undef FCNAME
 #define FCNAME MPL_QUOTE(FUNCNAME)
-MPL_STATIC_INLINE_PREFIX void MPIDI_CH4U_request_complete(MPIR_Request * req)
+MPL_STATIC_INLINE_PREFIX void MPIDIU_request_complete(MPIR_Request * req)
```

Comment:
Move this to `mpir_request.h` and MPIR namespace? It's not accessing any device information.

Reply:
Yes. I forgot to ask about this. I was a bit confuse about why we have this since we also have a `MPID_Request_complete`.

Reply:
Ah, right, we have `MPID_Request_complete`. I'm pretty sure we can replace all instances of this function with the `MPID` version. Can we try that instead? We can think about `MPIR` vs. `MPID` namespace later for that functionality later.

Reply:
I tried changing it to `MPID_Request_complete` and it has a failure in UCX embedded test. I think I will keep it for now and fix the problem later.

Reply:
OK, agreed that it shouldn't block this PR.